pkg/actions/commands.go

```diff
@@ -203,6 +203,19 @@ func Commands() {
 						return nil
 					},
 				},
+				{
+					Name:  "restart",
+					Usage: "Restart a single project, requires 'id' and 'startMode'",
+					Flags: []cli.Flag{
+						cli.StringFlag{Name: "id,i", Usage: "Project ID", Required: true},
+						cli.StringFlag{Name: "startmode, s", Usage: "Start Mode, either run or debug", Required: true},
```

Comment:
Cant see where startmode value is being checked.  The usage says it must be either 'run' or 'debug' but the request is dispatched to PFE without validation.  Also seems there is a 3rd allowed option `debugNoInit`.

Reply:
It gets checked in the request validation of PFE (400 error code if not), I didn't want to add validation in here too in case for example an extra startMode was added and we'd then have to make simultaneous changes. 

Good spot on debugNoInit, that is a 3rd valid mode yep.